include/bluetooth/mesh/gen_dtt_cli.h

```diff
@@ -111,15 +109,35 @@ int bt_mesh_dtt_get(struct bt_mesh_dtt_cli *cli, struct bt_mesh_msg_ctx *ctx,
  * unacknowledged message. Note that the response is a signed value, that can
```

Comment:
Update description for the out parameter when it is NULL.

Reply:
Did a proper run through the entire API description, there were several errors like this one. Should be fixed now